app/components-react/windows/settings/Multistreaming.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Services } from '../../service-provider';
+import { $t } from '../../../services/i18n';
+import { ObsSettingsSection } from './ObsSettings';
+import Translate from 'components-react/shared/Translate';
+import * as remote from '@electron/remote';
+
+export function MultistreamingSettings() {
+  const { UserService, MagicLinkService } = Services;
+
+  async function upgradeToPrime() {
+    const link = await MagicLinkService.getDashboardMagicLink('prime-marketing', 'slobs-ui-themes');
+    remote.shell.openExternal(link);
+  }
+
+  const shouldShowPrime = UserService.views.isLoggedIn && !UserService.views.isPrime;
+
+  // @@@ TODO: Add correct copy
```

Comment:
Are all the TODO's in this file still valid?  And are any of them blockers to merging this?

Reply:
Ah, I forgot to delete them - I'll cleanup